portal/src/test/scala/cromwell/pipeline/controller/UserControllerTest.scala

```diff
@@ -0,0 +1,111 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import cromwell.pipeline.datastorage.dto.{ UserDeactivationByEmailResponse, UserDeactivationByIdResponse, UserId }
+import cromwell.pipeline.service.UserService
+import cromwell.pipeline.utils.auth.SecurityDirective
+import cromwell.pipeline.{ AuthConfig, ExpirationTimeInSeconds }
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.{ Matchers, WordSpec }
+import pdi.jwt.JwtAlgorithm
+
+import scala.concurrent.Future
+
+class UserControllerTest extends WordSpec with Matchers with MockFactory with ScalatestRouteTest {
+
+  private val userService = stub[UserService]
+  private val userController = new UserController(userService)
+
+  "UserController" when {
+    "deactivateByEmail" should {
+      "return email and false value if user was successfully deactivated" in {
+        val deactivateUserByEmailRequest = "email"
+        val emailResponse = UserDeactivationByEmailResponse(email = "JohnDoe@cromwell.com", active = false)
```

Comment:
This is **really** confusing. After deactivation, we get a totally different email that we sent. Is there any explanation for this behavior?